.github/workflows/ci.yml

```diff
@@ -0,0 +1,124 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.5"
+          - "2.4"
+        puppet:
+          - "6"
+          - "5"
+        exclude:
+          - ruby: "2.5"
+            puppet: "5"
+          - ruby: "2.4"
+            puppet: "6"
+    env:
+      PUPPET_VERSION: "${{ matrix.puppet }}.0"
+    name: Puppet ${{ matrix.puppet }} (Ruby ${{ matrix.ruby }})
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-${{ matrix.puppet }}-${{ matrix.ruby }}-gems-unit-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.puppet }}-${{ matrix.ruby }}-gems-unit-
+      - name: Install dependencies
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle config without 'development system_tests release'
+          bundle install --jobs $(nproc) --retry 3
+      - name: Run tests
+        run: bundle exec rake
+
+  build_cache:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Enable IPv6 on docker
+        run: |
+          echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
+          sudo service docker restart
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: "2.7"
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-2.7-gems-acceptance-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-2.7-gems-acceptance-
+      - name: Install dependencies
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle config without 'development test release'
+          bundle install --jobs $(nproc) --retry 3
+
+  acceptance:
+    needs: build_cache
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        setfile:
+          - value: fedora32-64{hostname=fedora32-64.example.com}
+            name: Fedora 32
+          - value: fedora33-64{hostname=fedora33-64.example.com}
+            name: Fedora 33
+          - value: centos6-64{hostname=centos6-64.example.com}
+            name: CentOS 6
```

Comment:
CentOS 6 is going EOL in 2 weeks. Perhaps also time to drop it?

Reply:
fine by me. Did it in https://github.com/voxpupuli/puppet-borg/pull/81